db/migrate/20161109162634_add_origin_to_epic_queue_record.rb

```diff
@@ -0,0 +1,6 @@
+class AddOriginToEpicQueueRecord < ActiveRecord::Migration
+  def change
+    add_column :epic_queue_records, :origin, :string
+    add_column :epic_queue_records, :identity_id, :integer
```

Comment:
Should this be a reference instead? (http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)